katello_rapidnode.py

```diff
@@ -53,26 +56,56 @@ def cmd_debug(cmd):
 
 
 def paramiko_exec_command(system, username, password, command):
-    """Executes the command using paramiko"""
+    """Executes command using paramiko"""
     ssh = paramiko.SSHClient()
     ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
     ssh.connect(system, username=username, password=password)
     dummy, stdout, stderr = ssh.exec_command(command)
-    ret1 = stdout.read()
-    ret2 = stderr.read()
+    miko_stdout = stdout.read()
+    miko_stderr = stderr.read()
+    exit_status = stdout.channel.recv_exit_status()
     ssh.close()
-    return ret1, ret2
+    return miko_stdout, miko_stderr, exit_status
+
+
+def remote_cmd(host, username, password, command, halt_on_fail):
+    """Performs the actual paramiko goo and returns appropriate exit code"""
+    # std_results[0] = stdout
+    # std_results[1] = stderr
+    # std_results[2] = exit code
```

Comment:
these were actually comments to let people know what the values were, but I can delete that